components/Buttons/examples/ButtonsTypicalUse.m

```diff
@@ -134,30 +135,30 @@ - (void)viewDidLoad {
 
 - (void)didTap:(id)sender {
   NSLog(@"%@ was tapped.", NSStringFromClass([sender class]));
-  if (sender == self.floatingButton) {
-    [self.floatingButton
-          collapse:YES
-        completion:^{
-          dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(1 * NSEC_PER_SEC)),
-                         dispatch_get_main_queue(), ^{
-                           [self.floatingButton expand:YES completion:nil];
-                         });
-        }];
-  }
+//  if (sender == self.floatingButton) {
```

Comment:
Vas is dis? Commented out?

Reply:
Yah, just for now. I will need to migrate the animations to MDCButton.

Reply:
Please add a TODO above the comment here are below indicating that this is temporary pending the migration.

Reply:
Will-do when I migrate the example code.  This is the API review so I'm not committing this code.